apis/v1alpha1/validate.go

```diff
@@ -132,30 +132,17 @@ func (in *GreptimeDBCluster) validateFrontend() error {
 }
 
 func (in *GreptimeDBCluster) validateFrontendGroups() error {
-	nameCount := 0
 	for _, frontend := range in.GetFrontendGroups() {
-		if len(frontend.GetName()) != 0 {
-			nameCount++
+		// FIXME(liyang): The frontend name must be set when using frontendGroups.
```

Comment:
Why it needs `FIXME` here?

Reply:
I want to explain this name must be set, now it has been delete